helm/values-dev.yaml

```diff
@@ -120,7 +120,7 @@ microservice-chart:
     NPG_URI: "https://stg-ta.nexigroup.com/api/phoenix-0.0"
     NPG_READ_TIMEOUT: "10000"
     NPG_CONNECTION_TIMEOUT: "10000"
-    NPG_CARDS_PSP_LIST: "BCITITMM,CIPBITMM,BIC36019,UNCRITMM"
+    NPG_CARDS_PSP_LIST: "BCITITMM,CIPBITMM,BIC36019,UNCRITMM,BPPIITRRXXX,PPAYITR1XXX"
```

Comment:
Add this psp also to npg apim auth request policy for [dev env](https://github.com/pagopa/pagopa-infra/blob/a041699f82394744a007d90f5dc42c43b5ec5e86/src/domains/ecommerce-app/env/weu-dev/terraform.tfvars#L38)